CAP/gap/ConstructiveCategoriesRecord.gi

```diff
@@ -18,7 +18,9 @@ IsEnrichedOverCommutativeRegularSemigroup := Concatenation(
 IsAbCategory := Concatenation( [ 
   "ZeroMorphism", 
   "IsZeroForMorphisms", 
-  "AdditiveInverseForMorphisms" ], ~.IsEnrichedOverCommutativeRegularSemigroup ),
+  "SubtractionForMorphisms",
+  "AdditiveInverseForMorphisms",
+  "MultiplyWithElementOfCommutativeRingForMorphisms" ], ~.IsEnrichedOverCommutativeRegularSemigroup ),
```

Comment:
Again, this is the list for Ab-categories.
Thus, `"AdditiveInverseForMorphisms"` is correctly inserted here.